views/publication/summary/summary.templ

```diff
@@ -45,13 +45,51 @@ templ Summary(c *ctx.Ctx, args SummaryArgs) {
 								if args.Publication.Locked {
 									@views.BadgeLocked()
 								}
-								@BadgeType(c, args.Publication)
+                                <span class="c-subline ps-2 me-3 pe-3 border-end">
+                                    if args.Publication.Classification != "" {
+                                        { fmt.Sprintf("%s: %s", c.Loc.Get("publication_types." + args.Publication.Type), args.Publication.Classification) }
+                                    } else {
+                                        { c.Loc.Get("publication_types." + args.Publication.Type) }
+                                    }
+                                </span>
 								if mainFile := args.Publication.MainFile(); mainFile != nil {
-									@SummaryAccessLevel(c, mainFile)
+                                    <span class={ "c-subline", "me-3", "pe-3", templ.KV("border-end", mainFile.AccessLevel == "info:eu-repo/semantics/embargoedAccess") }>
+                                        if mainFile.AccessLevel == "info:eu-repo/semantics/openAccess" {
+                                            <i class="if if-download if--small if--success"></i>
+                                            <span class="c-subline text-truncate">{ c.Loc.Get("publication_file_access_levels." + mainFile.AccessLevel) }</span>
+                                        } else if mainFile.AccessLevel == "info:eu-repo/semantics/embargoedAccess" {
+                                            <i class="if if-time if--small if--muted"></i>
+                                            <span class="c-subline text-muted">{ c.Loc.Get("publication_file_access_levels." + mainFile.AccessLevel) }</span>
+                                        } else if mainFile.AccessLevel == "info:eu-repo/semantics/restrictedAccess" {
+                                            <i class="if if-ghent-university if--small if--primary"></i>
+                                            <span class="c-subline text-muted">{ c.Loc.Get("publication_file_access_levels." + mainFile.AccessLevel) }</span>
+                                        } else if mainFile.AccessLevel == "info:eu-repo/semantics/closedAccess" {
+                                            <i class="if if-eye-off if--small if--muted"></i>
+                                            <span class="c-subline text-muted">{ c.Loc.Get("publication_file_access_levels." + mainFile.AccessLevel) }</span>
+                                        }
```

Comment:
Could these access level literals be made a const in code somewhere or maybe boolean methods on file `file.IsOpenAccess()` instead of having them literally in templates?

Reply:
@verheyenkoen yes certainly, but in a later refactor that cleans up the vocabularies package